zapsyslog/core.go

```diff
@@ -47,6 +47,8 @@ func (core *Core) Write(entry zapcore.Entry, fields []zapcore.Field) error {
 		return errors.Wrap(err, "failed to encode log entry")
 	}
 
+	defer buffer.Free()
```

Comment:
Nice catch. I actually thought this is the buffer from stdlib 🙂

Reply:
I would just ask you to remove the empty line above this line and I am gonna merge this.

Reply:
It is a side-effect of using default `golangci-lint` config for a long time 😄

Reply:
@tchap done